tests/Doctrine/Tests/ORM/Functional/DatabaseDriverTest.php

```diff
@@ -230,6 +232,7 @@ private static function supportsUnsignedInteger(AbstractPlatform $platform): boo
         // NOTE: PostgreSQL and SQL SERVER do not support UNSIGNED integer
```

Comment:
This comment is no longer accurate. Please reword to eliminate the duplication with the actual code.